FishBunDemo/src/main/java/com/sangcomz/fishbundemo/WithFragmentActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.sangcomz.fishbundemo
+
+import android.os.Bundle
+import android.widget.RelativeLayout
+import androidx.appcompat.app.AppCompatActivity
+
+class WithFragmentActivity : AppCompatActivity() {
+
+    private lateinit var areaContainer: RelativeLayout
+    private lateinit var subFragment: SubFragment
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_withfragment)
+
+        areaContainer = findViewById(R.id.area_container)
+        subFragment = SubFragment()
```

Comment:
Fragment preferably call by `newInstance` static method even without parameter.
You can be helped in [Android Developer](https://developer.android.com/reference/android/app/Fragment)

Reply:
fixed it.

a854dd3